tests/DirectiveTest.php

```diff
@@ -3,14 +3,36 @@
 namespace Mazedlx\FeaturePolicy\Tests;
 
 use Mazedlx\FeaturePolicy\Directive;
+use Mazedlx\FeaturePolicy\Exceptions\InvalidDirective;
+use Mazedlx\FeaturePolicy\Value;
 
 final class DirectiveTest extends TestCase
```

Comment:
extracted the several use cases which the test covered. This allows us to be more specific in what we are testing.